### DISCUSSION TITLE
图片上传 Picture Uploading

### DISCUSSION THREAD

**Discussion by jianwang-beijing** [tamlok/vnote#1848] [General]
我记得还在2版本的时候有一个功能是: 右击图片, 选择上传到: Github/Gitee...
这个功能挺好的, 也方便吧md文件发送给别人可以浏览到图片, 这个功能希望3这个版本上加上

**Comment by tamlok** [comment]
https://github.com/vnotex/vnote/releases/tag/v3.6.0

**Reply by jianwang-beijing** [reply]
感谢快速的反馈, 但是我升级之后2的配置好像被修改了, 我现在能看到图床菜单, 但是默认是`本地`, 我不知道怎么配置其他的图床, 比如GitHub/Gitee, 请问怎么新增图床配置?

**Reply by tamlok** [reply]
看文档

**Reply by tamlok** [reply]
![image](https://user-images.githubusercontent.com/5763157/130926549-22d5d74c-ddef-4892-b2d0-f6e4cd82b06b.png)

**Reply by jianwang-beijing** [reply]
好的, 感谢
